<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkModuleProperties</class>
 <widget class="QDialog" name="LinkModuleProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Link text:</string>
     </property>
     <property name="buddy">
      <cstring>ELinkText</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="ELinkText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Link location:</string>
     </property>
     <property name="buddy">
      <cstring>ELinkLocation</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="ELinkLocation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="BChooseFile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>(leave it empty to remove link)</string>
     </property>
     <property name="buddy">
      <cstring>ELinkLocation</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>269</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="BRemove">
     <property name="toolTip">
      <string>Remove link</string>
     </property>
     <property name="text">
      <string comment="remove link">Remove</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <spacer name="spacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>161</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
